### DISCUSSION TITLE
Not Powering Up

### DISCUSSION THREAD

**Discussion by rockjock32** [STELLA-Landsat/STELLA#48] [Build Notes: How do I make a STELLA?]
New to soldering, so I'm not optimistic about the response, but I just completed a Stella 1.1 build and I cannot get it to power up. No lights come on at all. I verified the battery using a STELLA-Q I put together. I can say that the yellow LED comes on when I connect to my computer. I think the switch is working because the multimeter recognizes a connection between the two leads when it is on and not when it is off. Other than that I'm not really sure how to start troubleshooting connections. Any advice is appreciated!

**Comment by rockjock32** [comment]
Just a follow up, when I unmount the display, I get lights. So I guess it is either the display protoboard wiring or the head pin connectors.

**Reply by paulmirel** [reply]
1. Do you have a voltmeter/ multimeter? That'll set how we proceed. 
2. Clear photos of the boards showing the wiring are very helpful. We can do a lot of diagnoses with photos.

**Comment by rockjock32** [comment]
Hi Paul,
I do have a multimeter. And I just checked continuity between the wire loops in row 1 of the protoboard with the header by using a single double male pin. I also checked continuity between the headers and the solder points on the display side and all connections seem to be good. I might  a cross connection with the ground and + leads, but I don't know if that is an issue. Pics to follow

**Comment by rockjock32** [comment]
![IMG_7728](https://github.com/STELLA-Landsat/STELLA/assets/14187446/1626f55d-1b34-441b-b392-1f7e8035be42)
![IMG_7729](https://github.com/STELLA-Landsat/STELLA/assets/14187446/aee13010-3b7f-4535-a870-d4300a7cf257)

**Comment by rockjock32** [comment]
![IMG_7730](https://github.com/STELLA-Landsat/STELLA/assets/14187446/cb2fe704-1310-4241-acd4-ca3f709550d1)
![IMG_7731](https://github.com/STELLA-Landsat/STELLA/assets/14187446/1f257220-69d0-404c-8c9a-612643bdcfe8)

**Comment by paulmirel** [comment]
Looks to me like the red and black wires are touching at B1, B2 on the display board. You can check that with a meter set to continuity. Check the other wires along that line too, to make sure they aren't touching. You can pry them gently apart with a very small screwdriver, or redo the solder joint to that the wire passes the pin once, instead of wrapping around, to create some space.

**Comment by paulmirel** [comment]
Oops. B1, C1 is what I meant to say.

**Reply by rockjock32** [reply]
Yeah that is what the multimeter is telling me. I'll try to re-solder and report back. Definitely tight in there! Thanks for your time Paul.

**Reply by rockjock32** [reply]
That worked. It now powers up. Now on to the debugging!

**Comment by paulmirel** [comment]
Hooray!
